OemPkg/HelloUefi/readme.md

```diff
@@ -0,0 +1,19 @@
+# Simple Bootable Media Example
+
+Barebones example of how to boot to an EFI Application with no dependencies
+
+## To boot directly to this EFI application perform the following
```

Comment:
Is this step needed for systems that does not have UEFI shell? I am curious on the target users here.

Reply:
Yeah, initially the idea was to run this from UEFI Shell, but this change makes it simpler for someone without Shell or UEFI experience to verify it works.

Reply:
It may still be ran from the shell if desired but removes that dependency to see the print statement